### PR TITLE
Support traefik timeouts

### PR DESCRIPTION
**Pull request by minhdanh**
#### What this PR does / why we need it:

This PR adds support for traefik timeouts. https://docs.traefik.io/configuration/commons/#timeouts

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/traefik/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.65.0
+version: 1.65.1
```
Comment (cpanato) [inline stable/traefik/Chart.yaml]: please bump to since to is not a bug fix
```suggestion
version: 1.66.0
```

Comment (minhdanh) [reply stable/traefik/Chart.yaml]: Done

Comment (minhdanh) [reply stable/traefik/Chart.yaml]: @cpanato Can you have a look a the PR again?

**Comment by minhdanh** [comment]
@emilevauge @nmengin Can you have a look at the PR pls?

**Comment by dtomcej** [comment]
Hello @minhdanh, this looks really good! Can you please sign off on your commits, so that we can get this merged?

**Comment by minhdanh** [comment]
@dtomcej Done.

**Pull request merged**